src/components/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,200 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+import * as React from 'react';
+import { View, Animated } from 'react-native';
+
+import styled from 'styled-components/native';
+
+import { Paragraph } from 'components/Typography';
+import { spacing, fontStyles } from 'utils/variables';
+import { themedColors } from 'utils/themes';
+import { getDeviceWidth, reportLog } from 'utils/common';
+import { measure } from 'utils/ui';
+
+
+const screenWidth = getDeviceWidth();
+const ARROW_SIZE = 10;
+const TOOLTIP_WIDTH = 230;
+const SIDE_MARGIN = 20;
+
+type Layout = {
+  x: number,
+  y: number,
+  width: number,
+  height: number,
+};
+
+type Props = {
+  body: string,
+  positionOnBottom?: boolean,
+  isVisible: boolean,
+  wrapperStyle?: Object,
+  children?: React.Node,
+};
+
+const TooltipParagraph = styled(Paragraph)`
+  color: ${themedColors.control};
+  ${fontStyles.regular}px;
+  margin: 0;
+`;
+
+const TooltipWrapper = styled(Animated.View)`
+  width: ${TOOLTIP_WIDTH}px;
+  position: absolute;
+`;
+
+const TooltipArrowHolder = styled.View`
+  position: absolute;
+  width: ${ARROW_SIZE * 2}px;
+  height: ${ARROW_SIZE}px;
+  overflow: hidden;
+`;
+
+const TooltipArrow = styled.View`
+  width: ${ARROW_SIZE}px;
+  height: ${ARROW_SIZE}px;
+  border-radius: 2px;
+  transform: rotate(45deg);
+  background-color: #060f1ecc;
+  left: 4px;
+`;
+
+const BalloonWrapper = styled.View`
+  width: 100%;
+  margin: ${ARROW_SIZE}px 0;
+  align-items: center;
+`;
+
+const TooltipBalloon = styled.View`
+  background-color: #060f1ecc;
+  padding: ${spacing.mediumLarge}px;
+  border-radius: 14px;
+  align-items: center;
+`;
+
+const Tooltip = (props: Props) => {
+  const {
+    body,
+    children,
+    positionOnBottom = true,
+    isVisible,
+    wrapperStyle,
+  } = props;
+
+  const [wrapperLayout, setWrapperLayout] = React.useState<Layout>({
```

Comment:
https://flow.org/en/docs/react/children/ 
> Note: You need to use import * as React from 'react' here instead of import React from 'react' to get access to the React.Node type. We explain why that is in the React Type Reference.

Reply:
Aaaah, I see.
Didn't check that React.Node was needed :)